cli/openshift_deprecated.go

```diff
@@ -0,0 +1,28 @@
+package cli
```

Comment:
This file shouldn't be in cli. It should be in internal/cli now. I think you need to rebase.

Reply:
This will be removed. I did not rebase my code and there are old codes.